src/Compilers/Core/Portable/MetadataReader/MetadataDecoder.cs

```diff
@@ -2471,12 +2487,15 @@ internal bool DoesSignatureMatchEvent(TypeSymbol eventType, ParamInfo<TypeSymbol
             return !methodParam.IsByRef && methodParam.Type.Equals(eventType);
         }
 
+        [Flags]
         private enum AllowedRequiredModifierType
         {
-            None,
-            System_Runtime_CompilerServices_Volatile,
-            System_Runtime_InteropServices_InAttribute,
-            System_Runtime_InteropServices_UnmanagedType,
+            None = 0,
+            System_Runtime_CompilerServices_Volatile = 1,
+            System_Runtime_InteropServices_InAttribute = 1 << 1,
+            System_Runtime_InteropServices_UnmanagedType = 1 << 2,
+            System_Runtime_CompilerServices_IsExternalInit = 1 << 3,
```

Comment:
>System_Runtime_CompilerServices_IsExternalInit = 1 << 3, [](start = 12, length = 56)

Perhaps just leave a gap for this?

Reply:
I think if I left a gap it wouldn't merge cleanly, so I just copy/pasted to hopefully make the merge clean.

---
In reply to: [413415975](https://github.com/dotnet/roslyn/pull/43355#discussion_r413415975) [](ancestors = 413415975)

Reply:
> I think if I left a gap it wouldn't merge cleanly, so I just copy/pasted to hopefully make the merge clean.

I think it will still require minor manual intervention given the addition of a line below, pretty much the same level of intervention as for a gap. But I might be wrong.

---
In reply to: [413958941](https://github.com/dotnet/roslyn/pull/43355#discussion_r413958941) [](ancestors = 413958941,413415975)